apps/desktop/src/components/v3/BranchList.svelte

```diff
@@ -251,3 +344,32 @@
 </ReduxResult>
 
 <NewBranchModal {projectId} {stackId} bind:this={newBranchModal} />
+
+<Modal
+	bind:this={conflictResolutionConfirmationModal}
+	width="small"
+	defaultItem={{} as {
+		type: CommitStatusType;
+		commitId: string;
+		stackId: string;
+		hasConflicts: boolean;
+		isAncestorMostConflicted: boolean;
+		close: () => void;
+	}}
```

Comment:
Can this be made better? The only way to make a component generic is through a property.